utils/bazel/llvm-project-overlay/llvm/BUILD.bazel

```diff
@@ -3111,12 +3111,26 @@ cc_library(
 )
 
 cc_library(
-    name = "llvm-mca-headers",
+    name = "MCAApplication",
```

Comment:
Not sure this is the best name for this library. Open to suggestions. We can't just do what is done for `llvm-exegesis` as `MCA` is already taken by the actual MCA library, so it needs to be something else.